app/assets/stylesheets/global/african_poetics.scss

```diff
@@ -349,82 +368,96 @@ svg path.circle {
   stroke-width: 0.3;
 }
 
-// Additional style rules
-// The following are examples of colouring countries.  
-// You can substitute these with your own styles to colour 
-// the countries on the map.
-
-//Color a few small country circles (and the countries):
-
-//.ms, .ky {
-//  opacity:    1;
-//  fill:       #ff0000;
-// }
-
 // Remove european territories (i.e. Madeira, Reunion etc.)
 // Hiding these for now
+
 svg path.sh, 
 svg path.pt-30, 
 svg path.es-cn, 
 svg path.yt, 
 svg path.re {
   opacity: 0;
-  // fill: $brand-warning;
-  // fill-opacity:.5;
-  // stroke: $brand-warning;
 }
 
-// mapdemo page
-
-ul.country_list {
-  columns: 2;
-  font-size: .9em;
-  line-height: 1.2em;
-  list-style: none;
-  margin: 0 0 2em 0;
-  padding: 0;
-
-  li {
-    padding-left: 1.2em;
-    padding: .05em .1em;
-    text-indent: -1em;
-  } 
-}
+// Interactivity for Region Browse 
 
-.element_hover {
-  background-color: $nav-accent;
+.element_hover, .element_hover a {
+  background-color: $brand-danger;
+  color: white !important;
   cursor: pointer;
+  text-decoration: underline;
 }
 
 .country_highlight {
   fill-opacity: 1;
-  fill: red !important;
+  fill: $brand-danger !important;
   stroke-opacity: 1;
   stroke-width: 1;
-  stroke: red !important;
+  stroke: white !important;
 }
 
 .svg_hover {
   cursor: pointer;
-  fill: red !important;
+  fill: $brand-danger !important;
+  stroke: white;
 }
 
-#content-wrapper {
-  padding-top: 20px;
+// map display for regions
+// I could not narrow this down by page, possible fugure complication
+
+@media (min-width: $screen-phone) {
```

Comment:
Jess I wanted to call your attention to this potentially breaky code

Reply:
code will be moved to region specific page to avoid this